Public/Src/Engine/Processes/FileAccessReportLine.cs

```diff
@@ -0,0 +1,292 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.ContractsLight;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using BuildXL.Utilities;
+using BuildXL.Interop.MacOS;
+using BuildXL.Native.IO;
+using static BuildXL.Utilities.FormattableStringEx;
+using JetBrains.Annotations;
+
+namespace BuildXL.Processes
+{
+    /// <summary>
+    /// Low-level parser for file access report lines
+    /// </summary>
+    internal static class FileAccessReportLine

```

Comment:
Moved this out from SandboxedProcessReports to its own file. TryParse is left untouched. The only addition is GetReportLineForAugmentedFileAccess